### PR TITLE
.github: Add minimal issue template

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Cockpit version: xxx / future
```
Comment (martinpitt) [inline .github/issue_template.md]: Let's please add the OS here.

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+<!---
+If the issue you're experiencing is a bug, please help us out by explaining how to reproduce it.
+
+Other useful information to attach is a dump of the system log:
+`journalctl --since -10m` if the issue happened in the last 10 minutes
+`journalctl -u cockpit -u <unit>` if you know which unit it concerns
+-->
+
+Page: xxx <!--- for instance "/system" -->
```
Comment (martinpitt) [inline .github/issue_template.md]: That might be a bit confusing. How about just copy&pasting the URL here?

**Comment by Gundersanne** [comment]
example: https://github.com/Gundersanne/cockpit/issues/new/choose

image

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+name: Enhancement
+about: For feature requests and discussion of ideas
+title: Enhancement
+labels: 'enhancement'
+assignees: ''
+
+---
+
+Page: <!--- If applicable, for instance "http://localhost:9090/system" -->
```
Comment (larskarlitski) [inline .github/ISSUE_TEMPLATE/enhancement.md]: Sorry to comment on this one again, but I prefer if people just mentioned the name of the page ("Network", "Machines").

Comment (Gundersanne) [reply .github/ISSUE_TEMPLATE/enhancement.md]: https://github.com/cockpit-project/cockpit/pull/11042#discussion_r250276080 

I'm fine with either :smiley_cat:

Comment (martinpitt) [reply .github/ISSUE_TEMPLATE/enhancement.md]: Page name also WFM. It's usually clear from the description, so I almost feel it's redundant. Specific URL would be more helpful if you are on a particular subpage. But let's not block the PR on this, we can always change it later. So "Page:" it is.

**Pull request merged**